pom.xml

```diff
@@ -184,7 +184,7 @@
                         <plugin>
                             <groupId>com.qulice</groupId>
                             <artifactId>qulice-maven-plugin</artifactId>
-                            <version>0.15.4</version>
+                            <version>0.16.2</version>
```

Comment:
@vkuchyn can we upgrade to `0.16.3`? It is the latest one without performance problem